spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -165,6 +160,15 @@ public void open() {
     }
   }
 
+  private Map setupPySparkEnv() throws IOException{
+    Map env = EnvironmentUtils.getProcEnvironment();
+    if (!env.containsKey("PYTHONPATH")) {
+      SparkConf conf = getSparkConf();
+      env.put("PYTHONPATH", conf.get("spark.files").replaceAll(",", ":"));
```

Comment:
should this be `spark.submit.pyFiles` (or both `spark.files` and `spark.submit.pyFiles`)?
http://spark.apache.org/docs/latest/configuration.html#runtime-environment